sphinx/source/docs/user_guide/styling.rst

```diff
@@ -232,41 +232,55 @@ Color properties
 Bokeh objects have several properties related to colors. Use those color
 properties to control the appearance of lines, fills, or text, for example.
 
-Use one of these three options to define colors in Bokeh:
+Use one of these options to define colors in Bokeh:
+
+- Any of the
+  `named CSS colors <http://www.w3schools.com/colors/colors_names.asp>`_, such
+  as ``'green'`` or ``'indigo'``. You can also use the additional name
+  ``'transparent'`` (equal to ``'#00000000'``).
+- A CSS4 ``rgb()`` or ``rgba()`` color string, such as
+- ``'rgba(255, 0, 127, 0.6)'`` or ``'rgb(0 127 0 / 1.0)'``.
```

Comment:
I have tested CSS4 ``rgb()`` or ``rgba()`` color strings, and they seem to work both as individual values and as items in a list.

CSS4 also defines ``hsl()`` color strings - according to my tests, they only work as items in a list, but not as an individual color value.